### ISSUE TITLE
Friend Requests Play Sound

### ISSUE THREAD

**Issue by ImErinNeos** [Neos-Metaverse/NeosPublic#3136]
### Is your tweak request related to a problem? Please describe.

I never know when someone adds me until I open my dash

### Describe what would you like tweaked

Friend requests should have a proper notification that plays sound.

### Describe alternatives you've considered

none

**Comment by Dirko3000** [comment]
Yeah, the fact that there is no notification, in combination with you having to scroll down to find friend requests among your existing friends list is really annoying. this wouldn't even be as much of an issue if the friend requests were at the top of the list, or separated somehow, but I am aware that that has already been discussed elsewhere.

A notification would be fantastic. Having to actively search to see if you have any friend requests sucks.

**Comment by Banane9** [comment]
Yea, currently the easiest way to accept a friend request is to just open the requestee's profile from the session and accept from that
